shared/types/workspace.ts

```diff
@@ -5,7 +5,6 @@ import { UserId } from './user'
 export type WorkspaceId = Opaque<string, 'WorkspaceId'>
 
 export type Workspace = {
-  _id: WorkspaceId
```

Comment:
why remove this?

Reply:
Thought this was the wrong type to add the `_id` in. I've reverted this change